crates/dbsp/src/circuit/dbsp_handle.rs

```diff
@@ -490,15 +490,25 @@ enum Response {
 /// A handle to control the execution of a circuit in a multithreaded runtime.
 #[derive(Debug)]
 pub struct DBSPHandle {
-    // Time when the handle was created.
+    /// Time when the handle was created.
     start_time: Instant,
+
+    /// The underlying runtime.
+    ///
+    /// Normally this will be some runtime, but we take it out if we need to
```

Comment:
you mean, it may become None?

Reply:
Yes, when I say "take" here I'm referring to [Option::take](https://doc.rust-lang.org/stable/std/option/enum.Option.html#method.take)